contracts/staking/src/contract.rs

```diff
@@ -52,52 +52,64 @@ pub fn instantiate(
     msg: InstantiateMsg,
 ) -> Result<Response, ContractError> {
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
-    let operators = validate_addresses(&msg.operators, OSMOSIS_ACCOUNT_PREFIX)?;
-    let validators = validate_addresses(&msg.validators, CELESTIA_VALIDATOR_PREFIX)?;
 
     // TODO: determine if info.sender is the admin or if we want to pass in with msg
     ADMIN.set(deps.branch(), Some(info.sender.clone()))?;
 
-    if msg.ibc_channel_id == "" {
-        return Err(ContractError::ConfigWrong {});
-    }
-
-    if msg.native_token_denom == "" {
-        return Err(ContractError::ConfigWrong {});
-    }
-
-    validate_address(
-        &msg.multisig_address_config
-            .reward_collector_address
-            .to_string(),
-        &CELESTIA_ACCOUNT_PREFIX,
-    )?;
-    validate_address(
-        &msg.multisig_address_config.staker_address.to_string(),
-        &CELESTIA_ACCOUNT_PREFIX,
-    )?;
+    // validations
+    let validators = validate_addresses(&msg.validators, CELESTIA_VALIDATOR_PREFIX)?;
+    assert!(
+        msg.liquid_stake_token_denom.len() > 3,
+        "liquid_stake_token_denom is required"
+    );
+    assert!(
+        msg.liquid_stake_token_denom
+            .chars()
+            .all(|c| c.is_ascii_alphabetic()),
+        "liquid_stake_token_denom must be alphabetic"
+    );
 
     // Init Config
     let config = Config {
-        native_token_denom: msg.native_token_denom,
+        native_token_denom: "".to_string(),
         liquid_stake_token_denom: format!(
             "factory/{0}/{1}",
             env.contract.address, msg.liquid_stake_token_denom
         ), //TODO determine the format to save in
```

Comment:
Can we remove this TODO comment if it is resolved?

Reply:
```suggestion
        ),
```